### PR TITLE
Update doc for aarch64

### PR DESCRIPTION
**Pull request by wyzero**

### CODE REVIEW

**Comment by wyzero** [comment]
to #227

File: README.md [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
-  + [For TensorFlow Users](#for-tensorflow-users)
-  + [For PyTorch Users](#for-pytorch-users)
-* [Setup and Examples](#setup-and-examples)
-* [Publications](#publications)
-* [Tutorials and Documents for Developers](#tutorials-and-documents-for-developers)
-* [How to Contribute](#how-to-contribute)
-* [FAQ](#faq)
-  + [Roadmap with mlir-hlo Project](#roadmap-with-mlir-hlo-project)
-* [Contact Us](#contact-us)
+- [BladeDISC Introduction](#bladedisc-introduction)
```
Comment (JamesTheZ) [inline README.md:3]: Why do we want to include the index of the upper-level title inside the upper-level block.

Comment (wyzero) [reply README.md:3]: It's auto-generated by vs-code.

Comment (wyzero) [reply README.md]: Done, thanks!

**Review by JamesTheZ — APPROVED** [review]
LGTM.

**Pull request merged**